import-scripts/import-genie-data-transfer-deployment-color.sh

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env bash
+
+# import-genie-data-transfer-deployment-color.sh
+#
+# switch the (updated) standby genie web app deployments to become the new production genie web app deployments
+# and move the prior prodution deployment into a standby state
+#
+# the main required argument is the color ('green' or 'blue') which should become the new production deployment
+# the main steps executed will be:
+# - validate arguments
+# - scale down the replicas on the production deployment slightly, scale up the standy deployment. Allow time for pod readiness.
+# - scale down the replicas on the production deployment almost fully, scale up the standy deployment. Allow time for pod readiness.
+# - syncronize the genie user tables from the current production database to the current standby database
+# - clear the persistence caches for the current standby database
+# - switch the ingress rules to route website traffic to the incoming (standby until now) deployment
+# - mark the update process as complete in the management database table (implicitly changes user signup to the new production db)
+# - scale down the prior production deployment fully, scale up the new production deployment fully. Allow time for pod readiness.
+# - construct and check in to github repo the altered kubernetes configuration files
+
+
+FULL_REPLICA_COUNT=3
+unset GENIE_BLUE_DEPLOYMENT_LIST
+unset GENIE_GREEN_DEPLOYMENT_LIST
+declare -a GENIE_BLUE_DEPLOYMENT_LIST
+declare -a GENIE_GREEN_DEPLOYMENT_LIST
+GENIE_BLUE_DEPLOYMENT_LIST+=('cbioportal-backend-genie-public-blue')
+GENIE_BLUE_DEPLOYMENT_LIST+=('cbioportal-backend-genie-private-blue')
+GENIE_GREEN_DEPLOYMENT_LIST+=('cbioportal-backend-genie-public-green')
+GENIE_GREEN_DEPLOYMENT_LIST+=('cbioportal-backend-genie-private-green')
```

Comment:
just a thought, but maybe the deployment lists could be generalized/passed as arguments/listed in a config file somewhere? (Assuming that we will use this script in the future for non-genie deployments)